[1788.562536240] 2019-07-16 19:08:18,822 - INFO - launcher: Starting sanity checks
[1812.555074692] 2019-07-16 19:08:18,846 - INFO - launcher: Checking for Python 3.5+
[1812.555074692] 2019-07-16 19:08:18,846 - INFO - launcher: Checking console encoding
[1812.555074692] 2019-07-16 19:08:18,846 - INFO - launcher: Setting console encoding to UTF-8
[1812.555074692] 2019-07-16 19:08:18,846 - INFO - launcher: Ensuring we're in the right environment
[1844.547748566] 2019-07-16 19:08:18,878 - INFO - launcher: Adding local bins/ folder to path
[17887.321949005] 2019-07-16 19:08:34,921 - INFO - launcher: Required checks passed.
[17887.321949005] 2019-07-16 19:08:34,921 - INFO - launcher: Optional checks passed.
[17891.319990158] 2019-07-16 19:08:34,925 - INFO - launcher: Moving old musicbot log

######################### PRE-RUN SANITY CHECKS PASSED #########################

[56.1142587661743164] 2019-07-16 19:09:13,148 - WARNING - musicbot.config | In config.py::MainThread(12764), line 120 in run_checks: i18n file does not exist. Trying to fallback to config/i18n/en.json.
[56.1622459888458252] 2019-07-16 19:09:13,196 - INFO - musicbot.config | In config.py::MainThread(12764), line 131 in run_checks: Using i18n: config/i18n/en.json
[56.2342379093170166] 2019-07-16 19:09:13,268 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 226 in _setup_logging: Set logging level to INFO
[56.4581711292266846] 2019-07-16 19:09:13,492 - DEBUG - musicbot.json | In json.py::MainThread(12764), line 8 in __init__: Init JSON obj with config/i18n/en.json
[60.1825540065765381] 2019-07-16 19:09:17,216 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 91 in __init__: Starting MusicBot release-120519
[60.1825540065765381] 2019-07-16 19:09:17,216 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 97 in __init__: Loaded autoplaylist with 2531 entries
[64.9107894897460938] 2019-07-16 19:09:21,944 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 949 in on_ready: Connection established, ready to go.
[64.9107894897460938] 2019-07-16 19:09:21,944 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 340 in _cache_app_info: Caching app info
[65.0107350349426270] 2019-07-16 19:09:22,044 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 771 in _scheck_ensure_env: Ensuring data folders exist
[65.6178431510925293] 2019-07-16 19:09:22,651 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 787 in _scheck_server_permissions: Checking server permissions
[65.6178431510925293] 2019-07-16 19:09:22,651 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 791 in _scheck_autoplaylist: Auditing autoplaylist
[65.6178431510925293] 2019-07-16 19:09:22,651 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 795 in _scheck_configs: Validating config
[65.6178431510925293] 2019-07-16 19:09:22,651 - DEBUG - musicbot.config | In config.py::MainThread(12764), line 228 in async_validate: Validating options...
[65.6178431510925293] 2019-07-16 19:09:22,651 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 798 in _scheck_configs: Validating permissions config
[65.6178431510925293] 2019-07-16 19:09:22,651 - DEBUG - musicbot.permissions | In permissions.py::MainThread(12764), line 94 in async_validate: Validating permissions...
[65.6178431510925293] 2019-07-16 19:09:22,651 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 966 in on_ready: Connected: 596863422805573632/AntMusic#8286
[65.6218409538269043] 2019-07-16 19:09:22,655 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 974 in on_ready: Owner:     399325775200780300/Raresix#0020

[65.6218409538269043] 2019-07-16 19:09:22,655 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 977 in on_ready: Guild List:
[65.6218409538269043] 2019-07-16 19:09:22,655 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 981 in on_ready:  - AntNetwork Community
[65.6218409538269043] 2019-07-16 19:09:22,655 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 1024 in on_ready: Bound to text channels:
[65.6258394718170166] 2019-07-16 19:09:22,659 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 1025 in <listcomp>:  - AntNetwork Community/bot-commands
[65.6258394718170166] 2019-07-16 19:09:22,659 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 1050 in on_ready: Autojoining voice channels:
[65.6258394718170166] 2019-07-16 19:09:22,659 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 1051 in <listcomp>:  - AntNetwork Community/ðŸŽ¶  |  Music
[65.6258394718170166] 2019-07-16 19:09:22,659 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 279 in _join_startup_channels: Attempting to join AntNetwork Community/ðŸŽ¶  |  Music
[66.8655304908752441] 2019-07-16 19:09:23,899 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 731 in deserialize_queue: Deserializing queue for 501799014950371340
[69.1594207286834717] 2019-07-16 19:09:26,193 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 423 in get_player: Created player via deserialization for guild 501799014950371340 with 1 entries
[69.1594207286834717] 2019-07-16 19:09:26,193 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 295 in _join_startup_channels: Joined AntNetwork Community/ðŸŽ¶  |  Music
[69.1634202003479004] 2019-07-16 19:09:26,197 - DEBUG - musicbot.entry | In entry.py::MainThread(12764), line 56 in get_ready_future: Created future for audio_cache\youtube-e07zwkNoeLA-House_-_Au5_-_Follow_You_feat._Danyka_Nadeau_Monstercat_Release.webm
[69.1634202003479004] 2019-07-16 19:09:26,197 - FFMPEG - musicbot.player | In player.py::MainThread(12764), line 290 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-e07zwkNoeLA-House_-_Au5_-_Follow_You_feat._Danyka_Nadeau_Monstercat_Release.webm
[73.4920327663421631] 2019-07-16 19:09:30,526 - DEBUG - musicbot.player | In player.py::MainThread(12764), line 303 in _play: Playing <musicbot.player.SourcePlaybackCounter object at 0x000001A004EF5A90> using <discord.voice_client.VoiceClient object at 0x000001A004E6F240>
[73.5000324249267578] 2019-07-16 19:09:30,534 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 458 in on_player_play: Running on_player_play
[73.5040304660797119] 2019-07-16 19:09:30,538 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 707 in serialize_queue: Serializing queue for 501799014950371340
[333.8298397064208984] 2019-07-16 19:13:50,863 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 534 in on_player_finished_playing: Running on_player_finished_playing
[334.1041963100433350] 2019-07-16 19:13:51,138 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 707 in serialize_queue: Serializing queue for 501799014950371340
[334.1481707096099854] 2019-07-16 19:13:51,182 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 530 in on_player_stop: Running on_player_stop
[369.4305212497711182] 2019-07-16 19:14:26,464 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !play FWLR  Hot
[374.3763177394866943] 2019-07-16 19:14:31,410 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 1400 in cmd_play: {'_type': 'url', 'url': 'ytsearch:FWLR  Hot', 'ie_key': None, 'extractor': 'generic', 'webpage_url': 'FWLR  Hot', 'webpage_url_basename': 'FWLR  Hot', 'extractor_key': 'Generic'}
[378.6168551445007324] 2019-07-16 19:14:35,650 - DEBUG - musicbot.entry | In entry.py::MainThread(12764), line 56 in get_ready_future: Created future for None
[378.6208531856536865] 2019-07-16 19:14:35,654 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 622 in on_player_entry_added: Running on_player_entry_added
[378.6208531856536865] 2019-07-16 19:14:35,654 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 707 in serialize_queue: Serializing queue for 501799014950371340
[378.6563119888305664] 2019-07-16 19:14:35,690 - INFO - musicbot.entry | In entry.py::MainThread(12764), line 292 in _really_download: Download started: https://www.youtube.com/watch?v=mrCu4vr8RZc
[380.6156773567199707] 2019-07-16 19:14:37,649 - DEBUG - musicbot.entry | In entry.py::MainThread(12764), line 56 in get_ready_future: Created future for None
[381.7682564258575439] 2019-07-16 19:14:38,802 - INFO - musicbot.entry | In entry.py::MainThread(12764), line 302 in _really_download: Download complete: https://www.youtube.com/watch?v=mrCu4vr8RZc
[381.7812500000000000] 2019-07-16 19:14:38,815 - FFMPEG - musicbot.player | In player.py::MainThread(12764), line 290 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-mrCu4vr8RZc-FWLR_-_Hot_Monstercat_Release.webm
[382.1239907741546631] 2019-07-16 19:14:39,158 - DEBUG - musicbot.player | In player.py::MainThread(12764), line 303 in _play: Playing <musicbot.player.SourcePlaybackCounter object at 0x000001A00523B588> using <discord.voice_client.VoiceClient object at 0x000001A004E6F240>
[382.1269896030426025] 2019-07-16 19:14:39,161 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 458 in on_player_play: Running on_player_play
[382.1280086040496826] 2019-07-16 19:14:39,162 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 707 in serialize_queue: Serializing queue for 501799014950371340
[552.6912889480590820] 2019-07-16 19:17:29,725 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 534 in on_player_finished_playing: Running on_player_finished_playing
[552.9297897815704346] 2019-07-16 19:17:29,963 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 707 in serialize_queue: Serializing queue for 501799014950371340
[552.9377841949462891] 2019-07-16 19:17:29,971 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 530 in on_player_stop: Running on_player_stop
[4652.4528901576995850] 2019-07-16 20:25:49,487 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !Play fwlr hot
[4656.7135400772094727] 2019-07-16 20:25:53,747 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 1400 in cmd_play: {'_type': 'url', 'url': 'ytsearch:fwlr hot', 'ie_key': None, 'extractor': 'generic', 'webpage_url': 'fwlr hot', 'webpage_url_basename': 'fwlr hot', 'extractor_key': 'Generic'}
[4660.8716046810150146] 2019-07-16 20:25:57,905 - DEBUG - musicbot.entry | In entry.py::MainThread(12764), line 56 in get_ready_future: Created future for None
[4660.8766016960144043] 2019-07-16 20:25:57,910 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 622 in on_player_entry_added: Running on_player_entry_added
[4660.8766016960144043] 2019-07-16 20:25:57,910 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 707 in serialize_queue: Serializing queue for 501799014950371340
[4660.9207010269165039] 2019-07-16 20:25:57,954 - INFO - musicbot.entry | In entry.py::MainThread(12764), line 207 in _download: Download cached: https://www.youtube.com/watch?v=mrCu4vr8RZc
[4662.8790769577026367] 2019-07-16 20:25:59,913 - DEBUG - musicbot.entry | In entry.py::MainThread(12764), line 56 in get_ready_future: Created future for audio_cache\youtube-mrCu4vr8RZc-FWLR_-_Hot_Monstercat_Release.webm
[4662.8800141811370850] 2019-07-16 20:25:59,914 - FFMPEG - musicbot.player | In player.py::MainThread(12764), line 290 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-mrCu4vr8RZc-FWLR_-_Hot_Monstercat_Release.webm
[4662.9679174423217773] 2019-07-16 20:26:00,002 - DEBUG - musicbot.player | In player.py::MainThread(12764), line 303 in _play: Playing <musicbot.player.SourcePlaybackCounter object at 0x000001A004EBD358> using <discord.voice_client.VoiceClient object at 0x000001A004E6F240>
[4662.9849066734313965] 2019-07-16 20:26:00,019 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 458 in on_player_play: Running on_player_play
[4662.9879066944122314] 2019-07-16 20:26:00,022 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 707 in serialize_queue: Serializing queue for 501799014950371340
[4688.0555706024169922] 2019-07-16 20:26:25,089 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !help
[4714.2608008384704590] 2019-07-16 20:26:51,294 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !karaoke
[4743.9881434440612793] 2019-07-16 20:27:21,022 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !play fwlr hot
[4745.9650013446807861] 2019-07-16 20:27:22,999 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 1400 in cmd_play: {'_type': 'url', 'url': 'ytsearch:fwlr hot', 'ie_key': None, 'extractor': 'generic', 'webpage_url': 'fwlr hot', 'webpage_url_basename': 'fwlr hot', 'extractor_key': 'Generic'}
[4750.0729210376739502] 2019-07-16 20:27:27,107 - DEBUG - musicbot.entry | In entry.py::MainThread(12764), line 56 in get_ready_future: Created future for None
[4750.0759148597717285] 2019-07-16 20:27:27,110 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 622 in on_player_entry_added: Running on_player_entry_added
[4750.0759148597717285] 2019-07-16 20:27:27,110 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 707 in serialize_queue: Serializing queue for 501799014950371340
[4750.1130046844482422] 2019-07-16 20:27:27,147 - INFO - musicbot.entry | In entry.py::MainThread(12764), line 207 in _download: Download cached: https://www.youtube.com/watch?v=mrCu4vr8RZc
[4771.6002957820892334] 2019-07-16 20:27:48,634 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !pause
[4771.6092905998229980] 2019-07-16 20:27:48,643 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 525 in on_player_pause: Running on_player_pause
[92453.7461998462677002] 2019-07-17 20:49:10,780 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 2948 in on_guild_available: Guild "AntNetwork Community" has become available.
[92455.7477042675018311] 2019-07-17 20:49:12,781 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 949 in on_ready: Connection established, ready to go.
[92455.7477042675018311] 2019-07-17 20:49:12,781 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 954 in on_ready: Received additional READY event, may have failed to resume
[149468.6558749675750732] 2019-07-18 12:39:25,689 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 2948 in on_guild_available: Guild "AntNetwork Community" has become available.
[149470.6382586956024170] 2019-07-18 12:39:27,672 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 949 in on_ready: Connection established, ready to go.
[149470.6382586956024170] 2019-07-18 12:39:27,672 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 954 in on_ready: Received additional READY event, may have failed to resume
[154419.3856296539306641] 2019-07-18 14:01:56,419 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 940 in on_resumed: 
Reconnected to discord.

[163705.2781093120574951] 2019-07-18 16:36:42,312 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 940 in on_resumed: 
Reconnected to discord.

[164654.6920042037963867] 2019-07-18 16:52:31,726 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 940 in on_resumed: 
Reconnected to discord.

[169994.8800973892211914] 2019-07-18 18:21:31,914 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 2948 in on_guild_available: Guild "AntNetwork Community" has become available.
[169996.5715463161468506] 2019-07-18 18:21:33,605 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 949 in on_ready: Connection established, ready to go.
[169996.5715463161468506] 2019-07-18 18:21:33,605 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 954 in on_ready: Received additional READY event, may have failed to resume
[176906.0471770763397217] 2019-07-18 20:16:43,081 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 2948 in on_guild_available: Guild "AntNetwork Community" has become available.
[176908.0551021099090576] 2019-07-18 20:16:45,089 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 949 in on_ready: Connection established, ready to go.
[176908.0551021099090576] 2019-07-18 20:16:45,089 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 954 in on_ready: Received additional READY event, may have failed to resume
[177084.7525184154510498] 2019-07-18 20:19:41,786 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !help
[177092.6074306964874268] 2019-07-18 20:19:49,641 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !clear
[177105.7372972965240479] 2019-07-18 20:20:02,771 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !play tobymac scars
[177108.9718267917633057] 2019-07-18 20:20:06,005 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 1400 in cmd_play: {'_type': 'url', 'url': 'ytsearch:tobymac scars', 'ie_key': None, 'extractor': 'generic', 'webpage_url': 'tobymac scars', 'webpage_url_basename': 'tobymac scars', 'extractor_key': 'Generic'}
[177113.0901877880096436] 2019-07-18 20:20:10,124 - DEBUG - musicbot.entry | In entry.py::MainThread(12764), line 56 in get_ready_future: Created future for None
[177113.0931854248046875] 2019-07-18 20:20:10,127 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 622 in on_player_entry_added: Running on_player_entry_added
[177113.0941834449768066] 2019-07-18 20:20:10,128 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 707 in serialize_queue: Serializing queue for 501799014950371340
[177113.1701400279998779] 2019-07-18 20:20:10,204 - INFO - musicbot.entry | In entry.py::MainThread(12764), line 292 in _really_download: Download started: https://www.youtube.com/watch?v=yVj8DogtbVY
[177116.4705655574798584] 2019-07-18 20:20:13,504 - INFO - musicbot.entry | In entry.py::MainThread(12764), line 302 in _really_download: Download complete: https://www.youtube.com/watch?v=yVj8DogtbVY
[177128.0243141651153564] 2019-07-18 20:20:25,058 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !play
[177150.1173224449157715] 2019-07-18 20:20:47,151 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !help
[177171.0568225383758545] 2019-07-18 20:21:08,090 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !restart
[177171.3176221847534180] 2019-07-18 20:21:08,351 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 521 in on_player_resume: Running on_player_resume
[177176.537064075] launcher-INFO: Restarting in 2 seconds...
[177178.7944586277008057] 2019-07-18 20:21:15,828 - WARNING - musicbot.config | In config.py::MainThread(12764), line 120 in run_checks: i18n file does not exist. Trying to fallback to config/i18n/en.json.
[177178.8004138469696045] 2019-07-18 20:21:15,834 - INFO - musicbot.config | In config.py::MainThread(12764), line 131 in run_checks: Using i18n: config/i18n/en.json
[177178.8184347152709961] 2019-07-18 20:21:15,852 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 191 in _setup_logging: Skipping logger setup, already set up
[177178.8803999423980713] 2019-07-18 20:21:15,914 - DEBUG - musicbot.json | In json.py::MainThread(12764), line 8 in __init__: Init JSON obj with config/i18n/en.json
[177179.2708401679992676] 2019-07-18 20:21:16,304 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 91 in __init__: Starting MusicBot release-120519
[177179.2718250751495361] 2019-07-18 20:21:16,305 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 97 in __init__: Loaded autoplaylist with 2531 entries
[177181.8427672386169434] 2019-07-18 20:21:18,876 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 949 in on_ready: Connection established, ready to go.
[177181.8427672386169434] 2019-07-18 20:21:18,876 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 340 in _cache_app_info: Caching app info
[177181.9129321575164795] 2019-07-18 20:21:18,947 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 771 in _scheck_ensure_env: Ensuring data folders exist
[177182.0031392574310303] 2019-07-18 20:21:19,037 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 787 in _scheck_server_permissions: Checking server permissions
[177182.0041308403015137] 2019-07-18 20:21:19,038 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 791 in _scheck_autoplaylist: Auditing autoplaylist
[177182.0041308403015137] 2019-07-18 20:21:19,038 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 795 in _scheck_configs: Validating config
[177182.0051302909851074] 2019-07-18 20:21:19,039 - DEBUG - musicbot.config | In config.py::MainThread(12764), line 228 in async_validate: Validating options...
[177182.0051302909851074] 2019-07-18 20:21:19,039 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 798 in _scheck_configs: Validating permissions config
[177182.0051302909851074] 2019-07-18 20:21:19,039 - DEBUG - musicbot.permissions | In permissions.py::MainThread(12764), line 94 in async_validate: Validating permissions...
[177182.0061292648315430] 2019-07-18 20:21:19,040 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 966 in on_ready: Connected: 596863422805573632/AntMusic#8286
[177182.0081591606140137] 2019-07-18 20:21:19,042 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 974 in on_ready: Owner:     399325775200780300/Raresix#0020

[177182.0131237506866455] 2019-07-18 20:21:19,047 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 977 in on_ready: Guild List:
[177182.0151245594024658] 2019-07-18 20:21:19,049 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 981 in on_ready:  - AntNetwork Community
[177182.0181193351745605] 2019-07-18 20:21:19,052 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 1024 in on_ready: Bound to text channels:
[177182.0221216678619385] 2019-07-18 20:21:19,056 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 1025 in <listcomp>:  - AntNetwork Community/bot-commands
[177182.0241186618804932] 2019-07-18 20:21:19,058 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 1050 in on_ready: Autojoining voice channels:
[177182.0251181125640869] 2019-07-18 20:21:19,059 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 1051 in <listcomp>:  - AntNetwork Community/ðŸŽ¶  |  Music
[177182.0281140804290771] 2019-07-18 20:21:19,062 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 279 in _join_startup_channels: Attempting to join AntNetwork Community/ðŸŽ¶  |  Music
[177182.3166711330413818] 2019-07-18 20:21:19,350 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 731 in deserialize_queue: Deserializing queue for 501799014950371340
[177182.6664199829101562] 2019-07-18 20:21:19,700 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 423 in get_player: Created player via deserialization for guild 501799014950371340 with 2 entries
[177182.6674020290374756] 2019-07-18 20:21:19,701 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 295 in _join_startup_channels: Joined AntNetwork Community/ðŸŽ¶  |  Music
[177182.6684005260467529] 2019-07-18 20:21:19,702 - DEBUG - musicbot.entry | In entry.py::MainThread(12764), line 56 in get_ready_future: Created future for None
[177182.6694002151489258] 2019-07-18 20:21:19,703 - DEBUG - musicbot.entry | In entry.py::MainThread(12764), line 56 in get_ready_future: Created future for audio_cache\youtube-mrCu4vr8RZc-FWLR_-_Hot_Monstercat_Release.webm
[177182.6694002151489258] 2019-07-18 20:21:19,703 - FFMPEG - musicbot.player | In player.py::MainThread(12764), line 290 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-mrCu4vr8RZc-FWLR_-_Hot_Monstercat_Release.webm
[177184.2978129386901855] 2019-07-18 20:21:21,331 - DEBUG - musicbot.player | In player.py::MainThread(12764), line 303 in _play: Playing <musicbot.player.SourcePlaybackCounter object at 0x000001A006245A58> using <discord.voice_client.VoiceClient object at 0x000001A00627D748>
[177184.3028099536895752] 2019-07-18 20:21:21,336 - INFO - musicbot.entry | In entry.py::MainThread(12764), line 207 in _download: Download cached: https://www.youtube.com/watch?v=yVj8DogtbVY
[177184.3058073520660400] 2019-07-18 20:21:21,339 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 458 in on_player_play: Running on_player_play
[177184.3078057765960693] 2019-07-18 20:21:21,341 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 707 in serialize_queue: Serializing queue for 501799014950371340
[177199.2781026363372803] 2019-07-18 20:21:36,312 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !clear
[177202.4163784980773926] 2019-07-18 20:21:39,450 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !skip
[177202.5753123760223389] 2019-07-18 20:21:39,609 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 534 in on_player_finished_playing: Running on_player_finished_playing
[177202.7302408218383789] 2019-07-18 20:21:39,764 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 707 in serialize_queue: Serializing queue for 501799014950371340
[177202.7432374954223633] 2019-07-18 20:21:39,777 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 530 in on_player_stop: Running on_player_stop
[177214.2463026046752930] 2019-07-18 20:21:51,280 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !play tobymac scars
[177216.7328164577484131] 2019-07-18 20:21:53,766 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 1400 in cmd_play: {'_type': 'url', 'url': 'ytsearch:tobymac scars', 'ie_key': None, 'extractor': 'generic', 'webpage_url': 'tobymac scars', 'webpage_url_basename': 'tobymac scars', 'extractor_key': 'Generic'}
[177220.4124059677124023] 2019-07-18 20:21:57,446 - DEBUG - musicbot.entry | In entry.py::MainThread(12764), line 56 in get_ready_future: Created future for None
[177220.4134058952331543] 2019-07-18 20:21:57,447 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 622 in on_player_entry_added: Running on_player_entry_added
[177220.4144051074981689] 2019-07-18 20:21:57,448 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 707 in serialize_queue: Serializing queue for 501799014950371340
[177220.4273962974548340] 2019-07-18 20:21:57,461 - INFO - musicbot.entry | In entry.py::MainThread(12764), line 207 in _download: Download cached: https://www.youtube.com/watch?v=yVj8DogtbVY
[177222.4040539264678955] 2019-07-18 20:21:59,438 - DEBUG - musicbot.entry | In entry.py::MainThread(12764), line 56 in get_ready_future: Created future for audio_cache\youtube-yVj8DogtbVY-TobyMac_-_Scars_Come_With_Livin.webm
[177222.4050245285034180] 2019-07-18 20:21:59,439 - FFMPEG - musicbot.player | In player.py::MainThread(12764), line 290 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-yVj8DogtbVY-TobyMac_-_Scars_Come_With_Livin.webm
[177222.4784057140350342] 2019-07-18 20:21:59,512 - DEBUG - musicbot.player | In player.py::MainThread(12764), line 303 in _play: Playing <musicbot.player.SourcePlaybackCounter object at 0x000001A0062417F0> using <discord.voice_client.VoiceClient object at 0x000001A00627D748>
[177222.4814064502716064] 2019-07-18 20:21:59,515 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 458 in on_player_play: Running on_player_play
[177222.4834098815917969] 2019-07-18 20:21:59,517 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 707 in serialize_queue: Serializing queue for 501799014950371340
[177361.0085120201110840] 2019-07-18 20:24:18,042 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !disconnect
[177364.3366506099700928] 2019-07-18 20:24:21,370 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !pause
[177364.3436450958251953] 2019-07-18 20:24:21,377 - ERROR - musicbot.bot | In bot.py::MainThread(12764), line 2809 in on_message: Error in pause: CommandError: The bot is not in a voice channel.  Use !summon to summon it to your voice channel.
Traceback (most recent call last):
  File "C:\Users\CharlieR\Documents\GitHub\MusicBot\MusicBot\musicbot\bot.py", line 2708, in on_message
    handler_kwargs['player'] = await self.get_player(message.channel)
  File "C:\Users\CharlieR\Documents\GitHub\MusicBot\MusicBot\musicbot\bot.py", line 431, in get_player
    'Use %ssummon to summon it to your voice channel.' % self.config.command_prefix)
musicbot.exceptions.CommandError: The bot is not in a voice channel.  Use !summon to summon it to your voice channel.
[177367.5415973663330078] 2019-07-18 20:24:24,575 - INFO - musicbot.bot | In bot.py::MainThread(12764), line 2680 in on_message: 399325775200780300/Raresix#0020: !shutdown
[177369.5873210430145264] 2019-07-18 20:24:26,621 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 855 in safe_delete_message: Cannot delete message "!pause", message not found
[180241.1324906349182129] 2019-07-18 21:12:18,166 - ERROR - musicbot.bot | In bot.py::MainThread(12764), line 937 in on_error: Exception in on_message
Traceback (most recent call last):
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\site-packages\discord\client.py", line 270, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\CharlieR\Documents\GitHub\MusicBot\MusicBot\musicbot\bot.py", line 2839, in on_message
    await self.safe_delete_message(message, quiet=True)
  File "C:\Users\CharlieR\Documents\GitHub\MusicBot\MusicBot\musicbot\bot.py", line 849, in safe_delete_message
    return await message.delete()
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\site-packages\discord\message.py", line 697, in delete
    await self._state.http.delete_message(self.channel.id, self.id)
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\site-packages\discord\http.py", line 158, in request
    async with self.__session.request(method, url, **kwargs) as r:
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\site-packages\aiohttp\client.py", line 1005, in __aenter__
    self._resp = await self._coro
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\site-packages\aiohttp\client.py", line 497, in _request
    await resp.start(conn)
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\site-packages\aiohttp\client_reqrep.py", line 857, in start
    self._continue = None
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\site-packages\aiohttp\helpers.py", line 585, in __exit__
    raise asyncio.TimeoutError from None
concurrent.futures._base.TimeoutError
[180245.6872768402099609] 2019-07-18 21:12:22,721 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 2948 in on_guild_available: Guild "AntNetwork Community" has become available.
[180247.6713459491729736] 2019-07-18 21:12:24,705 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 949 in on_ready: Connection established, ready to go.
[180247.6713459491729736] 2019-07-18 21:12:24,705 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 954 in on_ready: Received additional READY event, may have failed to resume
[314333.6196651458740234] 2019-07-20 10:27:10,653 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 2948 in on_guild_available: Guild "AntNetwork Community" has become available.
[314335.5983011722564697] 2019-07-20 10:27:12,632 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 949 in on_ready: Connection established, ready to go.
[314335.5983011722564697] 2019-07-20 10:27:12,632 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 954 in on_ready: Received additional READY event, may have failed to resume
[318128.0282666683197021] 2019-07-20 11:30:25,062 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 2948 in on_guild_available: Guild "AntNetwork Community" has become available.
[318130.0295667648315430] 2019-07-20 11:30:27,063 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 949 in on_ready: Connection established, ready to go.
[318130.0295667648315430] 2019-07-20 11:30:27,063 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 954 in on_ready: Received additional READY event, may have failed to resume
[343875.9561550617218018] 2019-07-20 18:39:32,990 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 2948 in on_guild_available: Guild "AntNetwork Community" has become available.
[343877.9574961662292480] 2019-07-20 18:39:34,991 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 949 in on_ready: Connection established, ready to go.
[343877.9574961662292480] 2019-07-20 18:39:34,991 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 954 in on_ready: Received additional READY event, may have failed to resume
[368808.7411746978759766] 2019-07-21 01:35:05,775 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 2948 in on_guild_available: Guild "AntNetwork Community" has become available.
[368810.7418394088745117] 2019-07-21 01:35:07,775 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 949 in on_ready: Connection established, ready to go.
[368810.7418394088745117] 2019-07-21 01:35:07,775 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 954 in on_ready: Received additional READY event, may have failed to resume
[691653.7214314937591553] 2019-07-24 19:15:50,755 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 2948 in on_guild_available: Guild "AntNetwork Community" has become available.
[691655.7079603672027588] 2019-07-24 19:15:52,742 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 949 in on_ready: Connection established, ready to go.
[691655.7079603672027588] 2019-07-24 19:15:52,742 - DEBUG - musicbot.bot | In bot.py::MainThread(12764), line 954 in on_ready: Received additional READY event, may have failed to resume
